keps/sig-api-machinery/20190802-dynamic-coordinated-storage-version.md

```diff
@@ -172,6 +172,24 @@ During bootstrap, for each resource, the API server
   to avoid conflicting with other API servers.
 * installs the resource handler.
 
+The above mentioned process requires an API server to update the storageVersion
+before accepting API requests. If we don't enforce this order, data encoded in
```

Comment:
it seems like we should avoid reporting ready until this has been done.  Is it structured as a post-starthook?

Reply:
Yes. I need to add a post-starthook as well.

Note that using a post-start hook alone won't be enough, because the post-start hooks are asynchronous, which means it does not block write requests from success.